prisma/schema.prisma

```diff
@@ -12,18 +12,21 @@ datasource db {
 }
 
 model User {
-  id          Int         @id @default(autoincrement())
-  username    String      @unique
-  password    String
-  createdAt   DateTime    @default(now())
-  updatedAt   DateTime    @updatedAt
+  id        Int      @id @default(autoincrement())
+  username  String   @unique
+  password  String
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+  Movie     Movie[]
 }
 
 model Movie {
-  id            Int         @id @default(autoincrement())
-  title         String      @unique
-  description   String
-  runtimeMins   Int
-  createdAt     DateTime    @default(now())
-  updatedAt     DateTime    @updatedAt
+  id          Int      @id @default(autoincrement())
+  userId      Int
+  user        User     @relation(references: [id], fields: [userId])
```

Comment:
oooh i always put fields first, good to know it can be switched and not affect anything!

Reply:
found it my mistake and didnt bother to change it since it works! ahah